content/docs/capabilities/directory-sync.mdx

```diff
@@ -0,0 +1,54 @@
+---
+id: directory-sync
+title: Directory Sync
+description: Directory Sync in Pomerium Enterprise allows you to import organizational directory data and external data sources you can use in authorization policies.
+keywords: [directory sync, idp, identity provider, groups, directory]
+---
+
+# Directory Sync
+
+**Directory Sync** is the process of synchronizing external directory data from your identity provider into the Enterprise Console. This document discusses how directory sync works in Pomerium and its use cases.
+
+:::enterprise
+
+**Directory Sync** is a Pomerium Enterprise feature. [Contact us](https://www.pomerium.com/enterprise-sales/) to upgrade today.
+
+:::
+
+## Directory sync in the Enterprise Console
+
+To start a directory sync in the Enterprise Console:
```

Comment:
To me the phrase "To start a directory sync" suggests this is a one-time thing, rather than a recurring process. Maybe instead: "To enable directory sync"?